README.md

```diff
@@ -106,50 +128,48 @@ sql:
   - procedure/<schema_name>/<procedure_name>.sql
 ```
 
-**Note: Default basePath is `src/sql`.**
+## Usage
 
-3. Specify the base path of the folder for SQL files in `basePath` key.
+When installed globally, you can just invoke the CLI directly.
 
-4. List database objects in the order they need to be synced under the `sql` key.
+```bash
+$ sync-db
+```
 
-## Usage
+For local installation you can trigger it with [`npx`](https://www.npmjs.com/package/npx).
+```
+$ npx sync-db
+```
 
-Add `sync-db` script in your `package.json` file.
+### Using npm script
 
-### Add sync-db script in package.json
+You can also add a script into your project's `package.json` file like this:
 
-1. Add script in your `package.json`
 ```json
 {
   "scripts": {
-    "sync-db": "sync-db",
+    "sync-db": "sync-db"
   }
 }
 ```
 
-2. Run
+This allows you to trigger it like this:
```

Comment:
```diff
- it
+ sync-db
```

Reply:
This allows you to trigger `sync-db` from your project's root directory like this: